File: DY/DAY8/11655.py
# 11655번 ROT13
# https://www.acmicpc.net/problem/11655

for s in input():
    if s.islower(): # 소문자
        ch = ord(s)+13
        if ch > 122: # 소문자 범위를 벗어난 경우
            ch -= 26
        print(chr(ch), end='')
    elif s.isupper(): # 대문자
        ch = ord(s)+13
        if ch > 90: # 대문자 범위를 벗어난 경우
            ch -= 26
        print(chr(ch), end='')
    else: # 알파벳이 아닌 경우
        print(s, end='')

File: MY/DAY10/11655.java
/*
 * @author Minyeong Park
 * @date 2023.02.24.
 * ROT13
 * 문제 링크 : https://www.acmicpc.net/problem/11655
 */



import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        String str = br.readLine();
        for (int i = 0; i < str.length(); i++) {
            char c = str.charAt(i);
            if (c == ' ' || Character.isDigit(c)) {
                System.out.print(c);
            } else if (Character.isUpperCase(c)) {
                char ch = (char) (c + 13);
                if (ch > 'Z') {
                    ch = (char) (ch - 26);
                }
                System.out.print(ch);
            } else if (Character.isLowerCase(c)) {
                char ch = (char) (c + 13);
                if (ch > 'z') {
                    ch = (char) (ch - 26);
                }
                System.out.print(ch);
            }
        }
    }
}